raco/language/__init__.py

```diff
@@ -17,6 +17,19 @@ def opt_rules(self, **kwargs):
 class Language(object):
     __metaclass__ = ABCMeta
 
+    EQ = "=="
+    NEQ = "!="
+    PLUS = "+"
+    MINUS = "-"
+    DIVIDE = "/"
+    TIMES = "*"
+    IDIVIDE = "//"
+    GT = ">"
+    LT = "<"
+    GTEQ = ">="
+    LTEQ = "<="
+    MOD = "%"
+
```

Comment:
We already have default literals here https://github.com/uwescience/raco/blob/7c79f8f9adfb1b301106d4932748adb342678281/raco/expression/boolean.py#L52

Perhaps we should use those below instead of self.language.EQ, etc?